docs/settingup.md

```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
 }
 ```
 
+</details><details>
+<summary>subscribeobjects</summary>
```

Comment:
The actions are listed in alphabetical order, so the description of this action needs to be moved.

Reply:
This file is generated, thought we sorted automatically, maybe it's the order in "groups" which is used will try moving it there and re-generate

Reply:
seems to have been it